.github/2.0/cookbooks/Daemon.md

```diff
@@ -2,60 +2,159 @@
+    - [Run](#run)
+    - [API Docs](#api-docs)
+    - [JinaDClient](#jinadclient)
+      - [Connect from local](#connect-from-local)
+      - [Get remote status](#get-remote-status)
   - [Concepts](#concepts)
     - [Workspace](#workspace)
+      - [Special files](#special-files)
+      - [Create a workspace (redoc)](#create-a-workspace-redoc)
+      - [Get the details of a workspace (redoc)](#get-the-details-of-a-workspace-redoc)
+      - [List all workspaces (redoc)](#list-all-workspaces-redoc)
+      - [Delete a workspace (redoc)](#delete-a-workspace-redoc)
     - [RESTful Flows](#restful-flows)
-    - [RESTful Pods/Peas](#restful-podspeas)
-  - [Using JinaD](#using-jinad)
-    - [Run](#run)
-    - [Example Usage](#example-usage)
+      - [Create a Flow (redoc)](#create-a-flow-redoc)
+      - [Get details of a Flow (redoc)](#get-details-of-a-flow-redoc)
+      - [Terminate a Flow (redoc)](#terminate-a-flow-redoc)
+    - [RESTful Pods/Peas (Advanced)](#restful-podspeas-advanced)
+      - [Get all accepted arguments (redoc)](#get-all-accepted-arguments-redoc)
+      - [Create a Pea/Pod (redoc)](#create-a-peapod-redoc)
+      - [Get details of a Pea/Pod (redoc)](#get-details-of-a-peapod-redoc)
+      - [Terminate a Pea/Pod (redoc)](#terminate-a-peapod-redoc)
+    - [Logstreaming](#logstreaming)
     - [Development using JinaD](#development-using-jinad)
+        - [Build](#build)
+        - [Run](#run-1)
+        - [Why?](#why)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-# JinaD 2.0
+# JinaD (Daemon)
 
 JinaD is a daemon for deploying & managing Jina on remote via a RESTful interface. It achieves isolation of deployments by defining a `workspace` for each Jina object.
 
 ------
 
+## Using JinaD
+
+`JinaD` docker image is published on [Docker Hub](https://hub.docker.com/r/jinaai/jina/tags?page=1&ordering=last_updated&name=-daemon) along with core & follows the [standard image versioning](https://github.com/jina-ai/jina/blob/master/RELEASE.md#docker-image-versioning) used in Jina.
+
+### Run
+
+To deploy JinaD, SSH into a server (e.g.- ec2 instance) & run the below command.
+
+```bash
+docker run --add-host host.docker.internal:host-gateway \
+           -v /var/run/docker.sock:/var/run/docker.sock \
+           -v /tmp/jinad:/tmp/jinad \
+           -p 8000:8000 \
+           --name jinad \
+           -d jinaai/jina:master-daemon
+```
+
+**Note** : `JinaD` should always be deployed as a docker container. Simply starting the server using `jinad` command would not work.
+
+### API Docs
+
+- [Static docs with redoc](https://api.jina.ai/daemon/)
+
+- [Interactive swagger docs](http://localhost:8000/docs) (works once JinaD is started)
+
+### JinaDClient
+
+With v2.0.11, we introduce a Python Client to work with JinaD servers. It is async by default as it uses `aiohttp` internally. You can use `JinaDClient` or (for your async code) feel free to use `AsyncJinaDClient` which makes all following code awaitables.
```

Comment:
That paragraph is slightly confusing? You are saying that by default the client is async, but then there are async and sync versions to choose from? Maybe lets rather say that there exist sync and async versions and name them here as you do?